test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CommentsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
- [ ] You were requested not to use file generation command like scaffold command, but you used it anyways resulting in a lot of unnecessary files. Kindly remove this scaffold file.